styles/controls/progress-bar.scss

```diff
@@ -38,3 +38,17 @@
 		text-shadow: $progressbar-hudmeter-text-shadow;
 	}
 }
+
+.progressbar-circular {
+	width: 100px;
+	height: 100px;
+	box-shadow: $progressbar-shadow;
+
+	.CircularProgressBarFG {
+		//background-color: gradient(radial, 50% 50%, 0% 0%, 50%, 50%, from($light-blue), to($blue));
+	}
+
+	.CircularProgressBarBG {
+		background-color: $progressbar-background;
+	}
+}
```

Comment:
Interesting seeing use of this thing at last, but it's not actually being used anywhere in XML? Definitely seems like something that belongs in a separate commit at least

Reply:
Yea, I should have removed this. I started with this implementation and scrapped it. [progress-bar.scss](https://github.com/momentum-mod/panorama/pull/80/files/ce709da79e7d4ad3477bd6b321f6e509e04d7f47#diff-1f7f07fdfb6116dcd40601a66a312d733cfd4b8bad3fbef4ec0d003ea3f74eba) shouldn't even be modified.